content/help/ops/security/keys-and-certs/index.md

```diff
@@ -11,8 +11,8 @@ You can then verify that Citadel is actually generating keys and certificates:
 
 {{< text bash >}}
 $ kubectl get secret istio.my-sa -n my-ns
-NAME                                     TYPE            DATA      AGE
-istio.my-sa             istio.io/key-and-cert               3      24d
+NAME                    TYPE                           DATA      AGE
```

Comment:
Same as before, split the outputs from the commands. It applies to all instances but I am only commenting once more.

Reply:
That is in a different page and I would rather keep it as is. If you look at the istio.io you could find the command and output are combined together everywhere.

My PR touched a small part of that page to fix some indenting issues, but for detail content change, I want to avoid it in this PR.

Reply:
Yes, @geeknoid and I are discussing this now @yangminzhu 
I'm TAL now.